### ISSUE TITLE
All examples: feature conflict on Device::new with NVidia 2080 SUPER, driver 632.54, Win10

### ISSUE THREAD

**Issue by apessino** [vulkano-rs/vulkano#1621]
Hello! I am in the process of updating to 0.24 and noticed that, running *any* of the examples, I get the following panic on device creation:

```
Using device: GeForce RTX 2080 SUPER (type: DiscreteGpu)                                                                          
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: FeatureRestrictionNotMet(FeatureRestrictionError { feature
: "attachment_fragment_shading_rate", restriction: ConflictsFeature("shading_rate_image") })', examples\src\bin\triangle.rs:139:6 
```

Windows 10
GTX 2080 SUPER running the latest Studio Driver

Is this a know issue? Thanks...

A=

**Comment by Rua** [comment]
This should be fixed by #1614.

**Comment by apessino** [comment]
I will check that out - thanks for the quick response!

A=
